src/EasyPdf.php

```diff
@@ -176,15 +176,19 @@ public function loadHtml($html)
      */
     public function addImage($image, $x, $y, $width, $height)
     {
-        $this->pdf->Image($image, $x, $y, $width, $height);
+        if (str_contains(substr($image, -5), '.svg')) {
```

Comment:
`str_ends_with()`  is the new way

Reply:
i want to cover if have a last charakter like '/'